### PR TITLE
SM H->tautau Cards

### PR DESCRIPTION
**Pull request by thomas-mueller**
I would like to add `proc_card_mg5.dat` and `run_card.dat` files for NLO (just copied from existing gridpacks) and LO (new) ggH (->tautau) and VBF H (-> tautau). More details are found in the commit messages.

### CODE REVIEW

**Comment by thomas-mueller** [comment]
I would like to explain a bit more detailed, what our plans are, in the hope to get this PR merged as soon as possible. (I won't be able to work on this in the next 3 weeks.)

There are only Fall15/Spring16 Madgraph samples available for H->tautau (gluon fusion and VBF) events and I could not find any gridpack cards in this repository here.
```
dataset=/*HToTauTau*125*13TeV*amcatnlo*/*/MINIAODSIM
```
So the first aim was to get an updated Summer16 version of these samples.

For certain MC reweigting studies we need also leading order samples, for which I added cards that I created according to the descriptions in the two commit messages: https://github.com/cms-sw/genproductions/pull/1379/commits/531bd4dfff90c3be923cd963f49ac8f6f509cd3e, https://github.com/cms-sw/genproductions/pull/1379/commits/d9700b34d2dedaf80a1a4ca67145de0b688c45ee. If there are obvious problems in these settings, please let me know and I will try to fix them. I already created a gridpack with them and tested it successfully locally.

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SMHiggs/ggh012j_5f_LO_MLM_125_tarball/proc_card_mg5.dat [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model HC_NLO_X0_UFO-heft
+define p = g u c d s u~ c~ d~ s~
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SMHiggs/ggh012j_5f_LO_MLM_125_tarball/proc_card_mg5.dat]: please remove proton and jet definitions

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SMHiggs/ggh012j_5f_LO_MLM_125_tarball/proc_card_mg5.dat [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SMHiggs/ggh012j_5f_LO_MLM_125_tarball/proc_card_mg5.dat]: please remove lepton and neutrino definitions, as they are not needed/used

**Comment by thomas-mueller** [comment]
Done.

**Comment by efeyazgan** [comment]
SysCalc is not used anymore and these lines should be removed: 

+#**************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)

please also check https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by efeyazgan** [comment]
Please ignore the previous comment - I see only now that this is pre2017.

**Comment by perrozzi** [comment]
I think the "common" suggestions should be included, only the "for 2017 sample" should not be followed

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SMHiggs/ggh012j_5f_LO_MLM_125_tarball/proc_card_mg5.dat [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.2                 2014-11-06         *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SMHiggs/ggh012j_5f_LO_MLM_125_tarball/proc_card_mg5.dat]: please remove these banners as this is not anymore v2.2.2

**Comment by perrozzi** [comment]
any news?

**Pull request closed without merging**